hardware/detect_utils.py

```diff
@@ -282,7 +282,7 @@ def read_SMART_ata(hw, device, optional_flag="", mode=""):
         try:
             fields = line.split()
             if len(fields) < 10:
-                raise
+                raise ValueError('Number of fields lower than expected.')
```

Comment:
Could you be a bit more specific? "Expected 10 fields in %s, found %d" or something like that.

Reply:
I've added a reference to the line in the output and the number of fields